wwr/about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="A Rafting Company">
+    <title>White Water Rafting</title>
+    <link rel="stylesheet" href="about.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        rel="stylesheet">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <img src="images/logo.jpg" alt="logo" width="150" height="150">
+            <ul>
+                <a href="index.html">Index</a>
```

Comment:
A `ul` element should have only `li` children. Refer to this W3Schools link to learn more about `ul`:

https://www.w3schools.com/html/html_lists.asp#:~:text=Try%20it%20Yourself%20%C2%BB-,Unordered%20HTML%20List,-An%20unordered%20list

To add an `a` (anchor) tag inside a list, you should do the following:

```html
<ul>
  <li><a href="index.html">Index</a></li>
</ul>
```

![image](https://github.com/user-attachments/assets/2419e03c-78fd-4268-ab01-9530d14e0fe6)